src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
Yes. This one was a bit tricky. Basically we have a rule to only run examples if there are execute statements in the example. However, with the table execute command, the execute statement is part of the table header - so the "one example per row" examples were not executing. This method was my work-around on top of a work-around. I do actually prefer a marker interface in this case. However, I don't actually like marker interfaces because there is no way for a sub-class to remove the marker. I'll change it to Run and add JavaDoc.